README.md

```
@@ -17,14 +17,14 @@
 
 
 [![](https://img.shields.io/badge/Docs-GitHub%20wiki-brightgreen)](https://github.com/NLog/NLog/wiki)
-[![](https://img.shields.io/badge/Troubleshoot-Guide-orange)](https://github.com/nlog/nlog/wiki/Logging-troubleshooting)
+[![](https://img.shields.io/badge/Troubleshoot-Guide-orange)](https://github.com/NLog/NLog/wiki/Logging-troubleshooting)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)